### ISSUE TITLE
Updated ElectronicObserverEN to the latest version (Ver. 5.3.4), but it no longer starts.

### ISSUE THREAD

**Issue by yuki-kuzu** [ElectronicObserverEN/ElectronicObserver#455]
I have always found ElectronicObserverEN to be very useful. Thank you for everything.
I am using translation software to convert from Japanese. Please understand.

I started the software after a long time from my sub personal computer and updated ElectronicObserverEN to the latest version (ver. 5.3.4), but now it won't start.
The window of ElectronicObserverEN itself does not appear.

When I downgraded to the previous version (ver. 5.2.12), it started.
When I unzipped the latest version (ver. 5.3.4) that I downloaded and tried, it did not start, but the old version (ver. 5.2.12) that I downloaded did start.

The personal computer I am using is old, Windows 10 Pro, Core 2 Quad Q9650.
Is there any solution to this problem?
Thank you in advance for your help.

**Comment by myangelkamikaze** [comment]
It might be a problem with Microsoft Visual C++, try installing the latest version: https://aka.ms/vs/17/release/vc_redist.x64.exe
Details on what Microsoft Visual C++ is: https://learn.microsoft.com/ja-JP/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by yuki-kuzu** [comment]
Thank you for your response.
Unfortunately, I could not solve the problem.
I installed the latest version of Microsoft Visual C++ from the URL you provided and tried to start the latest version of ElectronicObserverEN (ver. 5.3.4) that I downloaded, but the symptoms remained the same and it would not start.

**Comment by myangelkamikaze** [comment]
Can you check if there's any errors in the Windows event viewer?
![unknown](https://github.com/ElectronicObserverEN/ElectronicObserver/assets/40002167/e5ccc5e5-0a02-45f0-9af7-88c81f5a3ca0)

**Comment by yuki-kuzu** [comment]
Thank you for your response.
This is for the most recent application log 256 lines.
Would this be okay?
Thank you in advance.

[Application Log.zip](https://github.com/ElectronicObserverEN/ElectronicObserver/files/14730625/Application.Log.zip)

**Comment by myangelkamikaze** [comment]
Okay, that's a strange error...
Can you try reinstalling the runtime: https://dotnet.microsoft.com/ja-jp/download/dotnet/thank-you/runtime-desktop-8.0.3-windows-x64-installer

**Comment by yuki-kuzu** [comment]
Thank you for your response.
I had tried to install runtime before launching the old version (ver. 5.2.12).
I tried to start the latest version (ver. 5.3.4) after installing runtime repair again, but the symptoms remained the same.
[Application Log3.zip](https://github.com/ElectronicObserverEN/ElectronicObserver/files/14730725/Application.Log3.zip)

**Comment by myangelkamikaze** [comment]
Yeah, it's still the same error.
Can you try restarting your computer?

**Comment by yuki-kuzu** [comment]
Yes, I rebooted after the runtime installation, but the symptoms did not change.

**Comment by myangelkamikaze** [comment]
Could you try downloading the latest version in some other location?
Also, maybe running it as administrator could help.

**Comment by yuki-kuzu** [comment]
Thank you for your repeated replies. I appreciate it.
I downloaded the file on another computer, transferred the file and tried to start it with administrator privileges, but no luck.
I will play with the old version for a while.

**Comment by myangelkamikaze** [comment]
5.2.12 is heavily outdated, so consider trying out the 岩川 version: https://github.com/dais-k/ElectronicObserver/releases

Yeah, I don't really have any ideas left. The error logs say that it's a .NET runtime error, so it's not really something we can fix.
